cryostatio/cryostat-web
# fix(dashboard): menus should appear over card container

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #884 

## Description of the change:

Mount menu to a higher DOM element. When dragging the card, the menus are closed by clicking on somewhere outside the card (e.g. page main container).

## Motivation for the change:

See #884 

## How to test

1. Run the test image.
2. Create an AA card.
3. Open a menu while card height is small (e.g used filter or collapse card).
4. Drag the card while menu is open (notice how the menu is closed).

## Screenshots

![Screenshot from 2023-04-12 22-50-40](https://user-images.githubusercontent.com/68053619/231635630-8eede12f-bb44-48d4-b3ed-d89ad41490ba.png)

![Screenshot from 2023-04-12 22-50-30](https://user-images.githubusercontent.com/68053619/231635635-fcfee522-600e-43f2-a056-2e0e1dd248d9.png)
